Tests/PostgreSQLTests/PostgreSQLConnectionTests.swift

```diff
@@ -332,6 +332,12 @@ class PostgreSQLConnectionTests: XCTestCase {
         print(row)
     }
 
+    func testRowCodableEmptyKeyed() throws {
```

Comment:
Can you add this to the `allTests` array at the bottom of the file?

Reply:
Oops. I always forget `allTests`. Fixed.